src/Elcodi/Admin/PageBundle/Resources/views/BlogPost/listComponent.html.twig

```diff
@@ -50,20 +50,18 @@
         {% endif %}
     </td>
 
-    {% include "@AdminCore/Tables/actions.html.twig" with {
-        entity: entity,
-        delete: not entity.persistent,
-        class: "page",
-    } %}
+    <td class="ta-c">
```

Comment:
This is not the same as calling the `actions` template with `{ entity: entity, class: 'blog_post' }`?